kalliope/core/ConfigurationManager/SettingLoader.py

```diff
@@ -499,8 +500,34 @@ def _get_default_synapse(settings):
         try:
             default_synapse = settings["default_synapse"]
             logger.debug("Default synapse: %s" % default_synapse)
-        except KeyError, e:
+        except KeyError:
             default_synapse = None
 
         return default_synapse
 
+    @staticmethod
+    def _get_settings_file_path():
+        """
+        used to load the settings.yml file
+        This function will try to load the file in this order:
+        - from the current directory where kalliope has been called. Eg: /home/me/Documents/kalliope_config
+        - from /etc/kalliope
+        - from the default settings.yml at the root of the project
+
+        :return: path to the settings.yml file
+        """
+        path_order = {
+            1: os.getcwd()+os.sep+FILE_NAME,
+            2: "/etc/kalliope"+os.sep+FILE_NAME,
+            3: utils.get_root_kalliope_path() + os.sep + FILE_NAME
+        }
+
+        for key in sorted(path_order):
+            file_path_to_test = path_order[key]
+            logger.debug("Try to load settings.yml file from %s: %s" % (key, file_path_to_test))
+            if os.path.isfile(file_path_to_test):
+                logger.debug("settings.yml file found in %s" % file_path_to_test)
+                return file_path_to_test
+
+        return None
```

Comment:
same that for brain : add argument for file path and test before doing the order